packages/okta-angular/src/okta/okta.service.ts

```diff
@@ -84,12 +84,20 @@ export class OktaAuthService {
     }
 
     /**
+     * Returns the operation to handle an unauthenticated state.
+     */
+    getOnAuthRequired(){
+      return this.config.getOnAuthRequired;
+    }
+
+    /**
      * Launches the login redirect.
      */
-    loginRedirect() {
+    loginRedirect(additionalParams?: object) {
       this.oktaAuth.token.getWithRedirect({
```

Comment:
yeah, here we can check if the responseType has a string in it, then split it into an array.

Reply:
Removing the ability to request a single token. Instead, we automatically return an `idToken` and `accessToken`